lib/routes.js

```diff
@@ -543,7 +543,7 @@ module.exports = function(self, options) {
       ], function(err) {
         if (err) {
           self.apos.utils.error(err);
-          res.status(500).send('error');
+          return res.status(500).send('error');
```

Comment:
This is duplicated in #258

Reply:
That branch has been merged and there are no conflicts, so resolving.